docs/user/reuse_garage_policy.md

```diff
@@ -0,0 +1,163 @@
+# Reusing a policy trained with Garage
```

Comment:
This will become the table-of-contents entry, so keep it terse

"Load and Use a Trained Policy"